pipeline/scraper/PSQL.py

```diff
@@ -30,10 +30,12 @@
     ON CONFLICT DO NOTHING;
     """
 
-user = os.environ['DB_USER'] or "postgres"
-password = os.environ['DB_PASSWORD'] or "secret"
-host = os.environ['DB_HOST'] or "localhost"
+user = os.environ.get('DB_USER') or "postgres"
```

Comment:
A bit cleaner approach would be `os.environ.get('DB_USER' , default = 'postgres')`